core/types/transaction.go

```diff
@@ -49,24 +48,28 @@ type txdata struct {
 	Recipient       *common.Address `rlp:"nil"` // nil means contract creation
 	Amount          *big.Int
 	Payload         []byte
-	V               byte     // signature
-	R, S            *big.Int // signature
+	V, R, S         *big.Int // signature
```

Comment:
what was the rationale behind changing V from `byte` to `big.Int`? sorry if i missed this in the Slack somewhere